### ISSUE TITLE
Is xnat still mounted on our file system?

### ISSUE THREAD

**Issue by edickie** [TIGRLab/admin#662]
The morning run scripts give us the error below...note how the * is not expanded...

ERROR: /mnt/xnat/spred/archive/ANDT/arc001/* folder is not named according to the data naming policy. Skipping

from franklin:
edickie@franklin:~$ ls /mnt/xnat/spred/archive/ANDT/arc001/*
ls: cannot access /mnt/xnat/spred/archive/ANDT/arc001/*: No such file or directory

Did the xnat mount move to another location?

**Comment by josephdviviano** [comment]
No,x
​nat might not be mounted on franklin. Is it mounted on tigrsrv?
​

JDV

**Comment by edickie** [comment]
I beleive not..           
> |_ .  _   _  _  _    
> |_ | (_) |  _) |  \/ 
>      _/              
> You have new mail.
> Last login: Tue Aug  2 15:02:29 2016 from mrsdti.camhres.ca
> edickie@tigrsrv:~$ df -h
> Filesystem                           Size  Used Avail Use% Mounted on
> udev                                  24G   12K   24G   1% /dev
> tmpfs                                4.8G  1.1M  4.8G   1% /run
> /dev/sda6                             92G   21G   67G  24% /
> none                                 4.0K     0  4.0K   0% /sys/fs/cgroup
> none                                 5.0M     0  5.0M   0% /run/lock
> none                                  24G  144K   24G   1% /run/shm
> none                                 100M   24K  100M   1% /run/user
> /dev/mapper/vg0-lv_backup             32T   29T  3.2T  90% /mnt/backup
> /dev/mapper/vg1-lv_scratch            33T   19T   13T  61% /mnt/scratch
> rprshnas01.camh.ca:/tigrlab/tigrlab  100T   49T   52T  49% /mnt/tigrlab
> pool                                 645G     0  645G   0% /pool
> edickie@tigrsrv:~$ ls /mnt/xnat/spred/archive/ANDT/arc001/* 
> ls: cannot access /mnt/xnat/spred/archive/ANDT/arc001/*: No such file or directory
> edickie@tigrsrv:~$

**Comment by josephdviviano** [comment]
I think IMG made changes to the firewall between XNAT and our system.

**Comment by josephdviviano** [comment]
I think this is a firewall issue. Looking into it now.

JDV

**Comment by josephdviviano** [comment]
I was able to resolve this by:

```
sudo apt-get remove --purge fail2ban ufw iptables
sudo apt-get install ufw iptables
```

I think fail2ban wasn't configured properly and keeps blocking us. Might be a good thing to have, but only if it is configured properly.

@edickie fyi, I no longer mount /mnt/xnat on all machines, only `tigrsrv`. we can change this back if you want, but I figured it would be cleaner this way seeing as the only real work happening in there is `clevis`.
